File: 1264-maximum-number-of-words-you-can-type/maximum-number-of-words-you-can-type.java
class Solution {
    public int canBeTypedWords(String text, String brokenLetters) {
        Set<Character> broken = new HashSet<>();
        for (char c : brokenLetters.toCharArray())
            broken.add(c);

        int count = 0;
        String[] words = text.split(" ");
        for (String w : words) {
            boolean valid = true;
            for (char c : w.toCharArray())
                if (broken.contains(c)) {
                    valid = false;
                    break;
                }

            if (valid)
                count++;
        }

        return count;
    }
}

File: 1264-maximum-number-of-words-you-can-type/maximum-number-of-words-you-can-type.go
func canBeTypedWords(text string, brokenLetters string) int {
	broken := make(map[rune]bool)
	for _, c := range brokenLetters {
		broken[c] = true
	}

	count := 0
	words := strings.Split(text, " ")
	for _, w := range words {
		valid := true
		for _, c := range w {
			if broken[c] {
				valid = false
				break
			}
		}

		if valid {
			count++
		}
	}
    
	return count
}